webpack.config.js

```diff
@@ -2,21 +2,29 @@ const path = require('path');
 const babelLoaderExcludeNodeModulesExcept = require('babel-loader-exclude-node-modules-except');
 
 module.exports = {
+  mode: 'production',
```

Comment:
this makes the dev process slower and I think breaks the ability to have stack traces from the TV emulator. 
When building the actual deployment version using package.sh I use `npm run prepare` which is already using the minimizer. If there is other benefit, maybe we should update there to call webpack with the mode changed.

Reply:
How is your workflow? What do you run for development purposes?
With `--mode`, the scripts can dynamically switch between production and development.

What would you set as default? Continuing the past configuration, I assume _development_ is the default and _production_ may be set with `npm run prepare`.

Reply:
`npm start` for dev

Reply:
See my last commit, 81bcf69, which takes care of this.